From 62a11cf698e948822752271a596466987b5c8e8a Mon Sep 17 00:00:00 2001
From: JP Sugarbroad <jpsugar@google.com>
Date: Thu, 19 Jul 2018 15:50:41 -0700
Subject: [PATCH 13/81] [RESTRICT AUTOMERGE] Remove build/libs

There's only a single library, libhost, and it's only used by acp and
atree in build/tools, move it there.

Bug: 109817231
Change-Id: Ifa58e58958d90fad008eb87a7eef4876b57b9edd
---
 {libs/host => tools/libhost}/Android.mk              | 0
 {libs/host => tools/libhost}/CopyFile.c              | 0
 {libs/host => tools/libhost}/include/host/CopyFile.h | 0
 3 files changed, 0 insertions(+), 0 deletions(-)
 rename {libs/host => tools/libhost}/Android.mk (100%)
 rename {libs/host => tools/libhost}/CopyFile.c (100%)
 rename {libs/host => tools/libhost}/include/host/CopyFile.h (100%)

diff --git a/libs/host/Android.mk b/tools/libhost/Android.mk
similarity index 100%
rename from libs/host/Android.mk
rename to tools/libhost/Android.mk
diff --git a/libs/host/CopyFile.c b/tools/libhost/CopyFile.c
similarity index 100%
rename from libs/host/CopyFile.c
rename to tools/libhost/CopyFile.c
diff --git a/libs/host/include/host/CopyFile.h b/tools/libhost/include/host/CopyFile.h
similarity index 100%
rename from libs/host/include/host/CopyFile.h
rename to tools/libhost/include/host/CopyFile.h
-- 
2.17.1

